postgres_scanner.cpp

```diff
@@ -397,13 +397,25 @@ static void PostgresInitInternal(ClientContext &context, const PostgresBindData
 		}
 		filter_string = " AND " + StringUtil::Join(filter_entries, " AND ");
 	}
+        if ( task_min == 0 && task_max == POSTGRES_TID_MAX ) {
+          // no use of CTID when task_min / task_max cover the largest range. 
+	  // This includes PostgreSQL compatible database which do not provide CTID (like YugabyteDB)
+          lstate.sql = StringUtil::Format(
+            R"(
+COPY (SELECT %s FROM "%s"."%s" WHERE 1=1                                          %s) TO STDOUT (FORMAT binary);
```

Comment:
So that you can remove "1=1" which is pretty ugly imo

Reply:
We need to keep WHERE only if there's a filter. I'll add that